utils/es.py

```diff
@@ -0,0 +1,19 @@
+from elasticsearch import Elasticsearch as Elasticsearch_official
+from elasticsearch import __version__ as es_pylib_version
+
+
+class Elasticsearch(Elasticsearch_official):
+    """Elasticsearch wrapper class
+
+    Wrapper class around the official Elasticsearch class that adds a simple version check upon initialization. In particular it checks if the major version of the library in use match the one of the cluster that we are tring to interact with. The check can be skip by setting to false the check_version parameter.
```

Comment:
I found it funny that a line that describes a _wrapper_ is not _wrapped_ to 72 columns :P

Reply:
:) Should be fixed now.